MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  MovieDetailsViewController.swift
+//  MovieApp
+//
+//  Created by endava-bootcamp on 28.03.2023..
+//
+
+import UIKit
+import PureLayout
+import MovieAppData
+
+class MovieDetailsViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate {
+    var movieBannerView: MovieBannerView!
+    var details: MovieDetailsModel?
+    var overviewLabel: UILabel!
+    var summary: UILabel!
+    var roleCollectionView: UICollectionView!
+    var crewList: [MovieCrewMemberModel]!
+    let reuseIdentifier = "cell"
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        details = MovieUseCase().getDetails(id: 111161)
+        guard let details else {return}
+        crewList = details.crewMembers
```

Comment:
Currently, it doesn't work if the details are nil, so it will be changed when the details are dynamically fetched and not hardcoded to be fetched from a single id.